src/packages/end-to-end/src/__tests__/api/postgres/query/null-filter.test.ts

```diff
@@ -0,0 +1,26 @@
+import request from 'supertest-graphql';
+import gql from 'graphql-tag';
+
+import { config } from '../../../../config';
+
+describe('null filter', () => {
+	test('should filter by Customers with company_null = true', async () => {
+		const { data } = await request<{ customers: any }>(config.baseUrl)
+			.query(gql`
+				query Customers($filter: CustomersListFilter) {
+					customers(filter: $filter) {
+						customerId
+						company
+					}
+				}
+			`)
+			.variables({
+				filter: {
+					company_null: true,
```

Comment:
Can we also add a similar test for `company_notnull: true` with an expected list of values?

Reply:
Yep, I'm just checking if the test passes first. Then I'll add more. But I don't see this test being run at all.